apps/src/p5lab/spritelab/CoreLibrary.js

```diff
@@ -366,12 +369,41 @@ export default class CoreLibrary {
     return speechBubbles[speechBubbles.length - 1];
   }
 
+  reachedSpriteMax() {
+    const numSprites = this.getNumberOfSprites();
+    return numSprites >= MAX_NUM_SPRITES;
+  }
+
+  // When the total number of sprites is equal to SPRITE_LIMIT_WARNING,
+  // a workspace alert warning is displayed
+  // Note that SPRITE_LIMIT_WARNING = MAX_NUM_SPRITES - 1
+  warnIfAtSpriteLimit() {
+    const numSprites = this.getNumberOfSprites();
+    if (numSprites === SPRITE_LIMIT_WARNING) {
```

Comment:
nit: same here as well (`if (this.getNumberOfSprites() === SPRITE_LIMIT_WARNING)`)

Reply:
Thanks! Made the update.